.github/workflows/build.yml

```diff
@@ -49,13 +49,23 @@ jobs:
 
       - name: Build Release
         run: cargo build --release
-
+      
+      - name: Build aarch64 for MacOS
+        if: ${{ runner.os == 'macOS' }}
+        run: |
+          rustup target add aarch64-apple-darwin
+          cargo build --release --target aarch64-apple-darwin
+          
       - name: Bundle macOS Release
         if: ${{ runner.os == 'macOS' }}
         run: |
           cargo install cargo-bundle
           cargo bundle --release
         working-directory: psst-gui
+      
+      - name: replacing binary with universal binary
```

Comment:
If this could be "Create macOS universal binary" to fit the tense and use of lipo, I think it would be good to merge!